cookbooks/cdo-varnish/libraries/http_cache.rb

```diff
@@ -139,6 +139,7 @@ def self.config(env)
               /amazon-future-engineer*
               /create-company-profile*
               /edit-company-profile*
+              /review-hociyskvuwa*
```

Comment:
For my learning, could you explain why this change is necessary?

Reply:
This is necessary because reviewing HOC events requires authentication - we need to know who you are and that you're a teacher. Adding the url to this list allows that information to be available on the Pegasus page.